pkg/controller/storagecluster/initialization_reconciler_test.go

```diff
@@ -450,6 +451,12 @@ func assertExpectedExternalResources(t assert.TestingT, reconciler ReconcileStor
 			for param, value := range expected.Data {
 				assert.Equal(t, value, actual.Parameters[param])
 			}
+			if actual.Name == "ocsinit-ceph-rgw" {
+				assert.NotEmpty(t, actual.Parameters["endpoint"], actual.Parameters["endpoint"])
+				assert.NotEmpty(t, actual.Parameters["region"], actual.Parameters)
+				assert.Equal(t, actual.Parameters["region"], "us-east-1")
+				assert.Equal(t, 3, len(actual.Parameters), actual.Parameters)
+			}
```

Comment:
Why are you making a special case for the RGW StorageClass? Isn't this already covered in the preceding for loop? If you're specifically checking for the absence of `objectStoreName`, you could just check to see if it exists in the array or not.

Reply:
> Why are you making a special case for the RGW StorageClass? Isn't this already covered in the preceding for loop?

We need a special case for RGW SC because it is different whether we do independent or converged mode.
> If you're specifically checking for the absence of `objectStoreName`, you could just check to see if it exists in the array or not.

Fixed!